Sources/KVKCalendar/DayView.swift

```diff
@@ -138,7 +138,7 @@ extension DayView: TimelineDelegate {
         components.hour = hour
         components.minute = minute
         let date = style.calendar.date(from: components)
-        return delegate?.willAddNewEvent(event, date) ?? event
+        return delegate?.willAddNewEvent(event, date)
```

Comment:
If the user does not implement the `delegate`, he will not be able to add new events, which is inconsistent with the previous behavior.

Perhaps you can consider returning the code to the original state... It is logically sound and I don't think it can be modified.

Reply:
```swift
guard let delegate else { return event }
return delegate.willAddNewEvent(event, date)
```

Reply:
the internal `delegate` isn't nil, so Im checking the external one in `KVKCalendarView+Extension.swift`